docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,40 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+AES Re-Encryption Tool:
```

Comment:
Association of a heading with its content is done by virtue of it being a heading - you don't need the <kbd>:</kbd>